### PR TITLE
Add VIA support to Noxary 220 and Update Noxary 280 for readability

### PR DESCRIPTION
**Pull request by Rozakiin**
## Description

- Adding VIA support for the Noxary 220
- Changing the Noxary 220 MCU name to atmega32u2 from atmega32u4
- Updating Noxary 280 readmes to conform to QMK template
- Updating Noxary 280 keymaps for readability of layouts
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/noxary/220/rules.mk [hunk 1]
```diff
@@ -30,6 +30,4 @@ UNICODE_ENABLE = no         # Unicode
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
-HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
-
-LAYOUTS = ortho_6x4
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
```
[the remark this answers is not in the document]
Comment (Rozakiin) [reply keyboards/noxary/220/rules.mk]: unable to commit suggestion, outdated comment?

File: keyboards/noxary/220/config.h [hunk 2]
```diff
@@ -20,12 +20,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x6060
+#define VENDOR_ID       0x4E58 //"NX"
+#define PRODUCT_ID      0x00DC //220
 #define DEVICE_VER      0x0001
 #define MANUFACTURER    Noxary
-#define PRODUCT         220
-#define DESCRIPTION     A custom numpad
+#define PRODUCT         Noxary 220
```
Comment (fauxpark) [inline keyboards/noxary/220/config.h:27]:
```suggestion
#define PRODUCT         220
```

File: keyboards/noxary/220/rules.mk [hunk 3]
```diff
@@ -31,5 +31,3 @@ BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
```
Comment (fauxpark) [inline keyboards/noxary/220/rules.mk:33]:
```suggestion

LAYOUTS = ortho_6x4
```

File: keyboards/noxary/280/config.h [hunk 4]
```diff
@@ -20,12 +20,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0x4E58
-#define PRODUCT_ID      0x0050
+#define VENDOR_ID       0x4E58 //"NX"
+#define PRODUCT_ID      0x0118 //280
 #define DEVICE_VER      0x0001
 #define MANUFACTURER    Noxary
-#define PRODUCT         280
-#define DESCRIPTION     A custom TKL
+#define PRODUCT         Noxary 280
```
Comment (fauxpark) [inline keyboards/noxary/280/config.h:27]:
```suggestion
#define PRODUCT         280
```

File: keyboards/noxary/280/readme.md [hunk 5]
```diff
@@ -1,9 +1,13 @@
-Noxary 280
+# Noxary 280
+
+![Noxary 280 WKL](https://i.imgur.com/CuQz0qW.png)
+![Noxary 280 WK](https://i.imgur.com/iAQYzsq.png)
```
Comment (noroadsleft) [inline keyboards/noxary/280/readme.md:4]: Swaps these images for smaller versions.

```suggestion
![Noxary 280 WKL](https://i.imgur.com/CuQz0qWl.png)
![Noxary 280 WK](https://i.imgur.com/iAQYzsql.png)
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**